### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by yahgwai**
* Combine the wiki docs with the redoc docs at docs.html
* write better overall description for appointment - including about the three sections
* only allow 0 refund
* change gas limit to integer and set limit eg 2000000
* only allow 1 or 0 for mode
* only allow pre and post condition to be 0x

### CODE REVIEW

File: src/dataEntities/appointment.ts [hunk 1]
```diff
@@ -55,14 +55,14 @@ export interface IAppointmentBase {
...
 
     /**
-     * How much to refund the customer by, in wei. Maximum can be 0.1 ether = 100000000000000000
+     * How much to refund the customer by, in wei. Currently set to 0.
      */
     readonly refund: string;
 
     /**
-     * The amount of gas to use when calling the external contract with the provided data. Maximum is 6 million = 6000000.
+     * The amount of gas to use when calling the external contract with the provided data. Maximum is 6 million = 2000000.
```
Comment (bigspider) [inline src/dataEntities/appointment.ts]: Typo "6 million = 2000000".

File: src/public/swagger-doc.json [hunk 2]
```diff
@@ -0,0 +1,182 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "PISA is an accountable watching service. PISA can be hired to watch for on-chain events by requesting an appointment. At it's core PISA is a watch and respond service, you can configure PISA to watch for EVM events and provide it with data to respond with if the event occurs.\n",
```
Comment (bigspider) [inline src/public/swagger-doc.json]: Typo "its core".

File: src/public/swagger-doc.json [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        "id": {
+          "type": "number",
+          "description": "A unique id, chosen by the customer.",
+          "minimum": 0,
+          "maximum": 9007199254740991,
+          "example": 200
+        },
+        "jobId": {
+          "type": "integer",
+          "description": "A counter used to replace appointments of the same id, but lower counter. A knew appoinitment can be requested for an existing id as long as the job id is incremented. If this is the case PISA will forget the old appointment and only watch for the new one.",
```
Comment (bigspider) [inline src/public/swagger-doc.json]: typos "knew appoinitment"

File: src/public/swagger-doc.json [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+          "minimum": 0,
+          "maximum": 9007199254740991,
+          "example": 3
+        },
+        "startBlock": {
+          "type": "integer",
+          "description": "The block at which this appointment starts. Only events that occur after (inclusive) start block and before (inclusive end block will be responded to.",
```
Comment (bigspider) [inline src/public/swagger-doc.json]: missing closing parenthesis.

File: src/dataEntities/appointment.ts (hunk 1 above)
Comment (yahgwai) [reply src/dataEntities/appointment.ts]: Done

File: src/public/swagger-doc.json (hunk 2 above)
Comment (yahgwai) [reply src/public/swagger-doc.json]: Done

File: src/public/swagger-doc.json (hunk 3 above)
Comment (yahgwai) [reply src/public/swagger-doc.json]: Done

File: src/public/swagger-doc.json (hunk 4 above)
Comment (yahgwai) [reply src/public/swagger-doc.json]: Done

**Pull request merged**